robottelo/vm.py

```diff
@@ -240,6 +241,13 @@ def install_katello_agent(self):
         result = self.run('rpm -q katello-agent')
         if result.return_code != 0:
             raise VirtualMachineError('Failed to install katello-agent')
+        if bz_bug_is_open('1431747'):
+            gofer_start = self.run('service goferd start')
+            if gofer_start.return_code != 0:
+                raise VirtualMachineError('Failed to start katello-agent')
+        gofer_check = self.run('service goferd status')
+        if gofer_check.return_code != 0:
+            raise VirtualMachineError('Failed to start katello-agent')
```

Comment:
you've just checked status so it would be better to say "Katello-agent is not running"